src/cleaner/config/cleaner.py

```diff
@@ -18,23 +18,29 @@
 
 import logging
 import logging.config
+import copy
 
+class Cleaner(object):
 
-class Cleaner:
-
-    def __init__(self, cluster_configuration, service_configuration, default_service_configuraiton):
+    def __init__(self, cluster_conf, service_conf, default_service_conf):
         self.logger = logging.getLogger(__name__)
-
-        self.cluster_configuration = cluster_configuration
+        self.cluster_conf = cluster_conf
+        self.service_conf = service_conf
+        self.default_service_conf = default_service_conf
```

Comment:
>default_service_conf [](start = 36, length = 20)

there is a cluster-object-model you should follow. Please ping Yundong for help.

Reply:
I followed other module's config code to do this. Confirmed with Yundong and the format is right. Will add Yundong to review when I finish other fixes.